chevah/github_hooks_server/handler.py

```diff
@@ -41,7 +32,7 @@ def dispatch(self, event):
         handler = getattr(self, event.name, None)
         if handler is None:
             message = f'No handler for "{event.name}"'
-            logging.error(message)
+            logging.warning(message)
```

Comment:
I think that this should be debug.

it's OK to not have handlers for a certain action.